cmd/fossa/build.go

```diff
@@ -4,26 +4,24 @@ import (
 	"fmt"
 	"os"
 
-	logging "github.com/op/go-logging"
 	"github.com/urfave/cli"
-)
 
-type buildConfig struct {
-	force bool
-}
+	config "github.com/fossas/fossa-cli/config"
```

Comment:
`config` doesn't need to be a named import. This seems to happen a lot, you might want to check if you have a misconfigured IDE or `goimports`.